bin/main/application-dev.yaml

```diff
@@ -0,0 +1,61 @@
+spring:
```

Comment:
What's the purpose of this `.yaml`, and how does its purpose differ from `src/main/resources/application-dev.yaml`?

Reply:
deleted